src/jumio-transactions/jumio-transaction.service.ts

```diff
@@ -58,9 +58,10 @@ export class JumioTransactionService {
   async findByWorkflowExecutionId(
     workflowExecutionId: string,
   ): Promise<JumioTransaction | null> {
-    return await this.prisma.jumioTransaction.findFirst({
+    const workflows = await this.prisma.jumioTransaction.findMany({
       where: { workflow_execution_id: workflowExecutionId },
     });
```

Comment:
We should use latest one if we somehow end up with duplicates (although this should never happen)